cli/ps.go

```diff
@@ -65,7 +67,12 @@ func (p *PsCommand) runPs(args []string) error {
 
 	if p.flagQuiet {
 		for _, c := range containers {
-			fmt.Println(c.ID[:6])
+			switch p.flagNoTrunc {
```

Comment:
how about
id := c.ID[:6] 
if p.flagNoTrunc {
    id = c.ID
}

fmt.Println(id)

This will make the code more clean，WDYT?

Reply:
Perfect. Thx :+1: